test/unit/sthDatabase_test.js

```diff
@@ -118,6 +118,19 @@ function connectToDatabase(callback) {
     );
 }
 
+/**
+ * Check DB status and return error 500 if DB is not connected
+ */
+function CheckDatabaseStatus() {
+    let client;
+    if (!client.isConnected()) {
```

Comment:
You shouldn't include conditional code for expectations (e.g. this "if" statement) in unit tests. You should prepare the pre-conditions of the tests in such way (in this case, having the DB unconnected) that expectations always hold. 

As a side-note (minor) function name should be in [camelCase](https://en.wikipedia.org/wiki/Camel_case) style, so `checkDatabaseStatus()` should be used